mono/metadata/assembly.c

```diff
@@ -133,36 +135,140 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	{"Novell.Directory.Ldap", 0},
 	{"PEAPI", 0},
 	{"System", 0},
+	{"System.AppContext", 0},
```

Comment:
because our facades have newer version numbers than what assemblies (such as roslyn) reference because they were built with an older version of .NET Framework than what we're pretending to be.  (that's my very sketchy understanding of how this works)